R/scope.R

```diff
@@ -389,3 +389,12 @@ renv_scope_caution <- function(value) {
     scope = parent.frame()
   )
 }
+
+renv_scope_verbose_if <- function(value) {
+  if (value) {
+    renv_scope_options(
+      renv.verbose = value,
+      scope = parent.frame()
+    )
+  }
+}
```

Comment:
```suggestion
renv_scope_verbose_if <- function(value, scope = parent.frame()) {
  if (value) {
    renv_scope_options(
      renv.verbose = TRUE,
      scope = scope
    )
  }
}
```

`value` -> `TRUE` makes the logic a bit clearer IMO and I think it's good practice to always have a` scope` argument.